docs/administration/security/project-acl.md

```diff
@@ -0,0 +1,39 @@
+# 3.4.0 ACL Changes
+
+## Project ACLs for Key Storage
+In Rundeck 3.4.0, the ACL policies were enhanced to allow administrators the ability to write project level ACLs for key storage access.  Previously, ACL policies for Key Storage Access were defined at the system level.  Now, administrators and authorized users can control key storage access at a project level in order to isolate users from other contexts at the System level. 
+
+The default behavior for Key Storage access is defined at the project level.  For administrators who would like to retain the previous functionality of defining key storage access at the system level, they would have to set the following config property:
+
+
+
+     rundeck.feature.projectKeyStorage.enabled=false
+
+
+
+This property should be set in the `rundeck-config.properties`. The Key Storage GUI is available in the project configuration menu allowing project level key management.  
+## Enterprise ACL Storage Layer
```

Comment:
This section doesn't make sense on this page.  It has nothing to do with Key Storage.